README.md

```diff
@@ -11,6 +11,12 @@ See [action.yml](action.yml) for a complete list of options and outputs.
 ```yaml
 steps:
 - uses: actions/checkout@v4
+  with:
+    # This action needs access to the history and tags.
+    # You can also limit the depth to a smaller number and run
```

Comment:
How can you go smaller than zero? Smaller than what?

Reply:
Improved.
done.